packer/install-dependencies.sh

```diff
@@ -266,11 +218,9 @@ done
 
 CORE_DNS_VERSIONS="
 1.6.5
```

Comment:
I see error in pull 1.6.6, so maybe we can baker that as well, since it's just 40MB.
mcr.microsoft.com/oss/kubernetes/coredns:1.6.6

Reply:
and 
mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.3.0_v0.0.5